### PR TITLE
Change to Valueset-rsg-type

### PR DESCRIPTION
**Pull request by mjosborne1**
for https://jira.hl7.org/browse/FHIR-46544
add SNOMED CT AU code 1515311000168102|Biological sex at birth|

### CODE REVIEW

**Comment by mjosborne1** [comment]
Please hold off - there seems to be a strange issue with the expansion of this ValueSet.

**Comment by mjosborne1** [comment]
OK, It's expanding now. Weird...I didn't change anything, but maybe it was terminology server related.

File: input/resources/valueset-rsg-type.xml [hunk 1]
```diff
@@ -33,6 +33,12 @@
         </include>    
         <include>
             <system value="http://terminology.hl7.org.au/CodeSystem/rsg-type" />
+        </include>
+        <include>
+            <system value="http://snomed.info/sct"/>
+            <concept>
+                <code value="1515311000168102" />
+            </concept>
         </include>
```
Comment (dbojicic) [inline input/resources/valueset-rsg-type.xml:42]: Commenting here to get an inline comment. 
Please update change log.

**Review by dbojicic — CHANGES_REQUESTED** [review]
One change requested.

**Review by dbojicic — APPROVED** [review]

**Review by brettesler-ext — APPROVED** [review]

**Pull request merged**